src/Components/Common/MentionDropdown.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
+import routes from "../../Redux/api";
+import useQuery from "../../Utils/request/useQuery";
+
+const MentionsDropdown: React.FC<{
+  onSelect: (user: any) => void;
```

Comment:
`any` types are not allowed. Change all usages of `any` types introduced in this PR to instead have proper types.

Reply:
got it